resources/js/components/Committee.vue

```diff
@@ -0,0 +1,65 @@
+<script>
+import CommitteeInfo from './CommitteeInfo';
+import CommitteeMemberList from './CommitteeMemberList';
+import CommitteeSessions from './CommitteeSessions';
+export default {
+    name: 'Committee',
+    props: {
+        info: {
+            type: String,
+            default: () => 'info'
```

Comment:
FYI for strings you could do just `default: 'info'`, so no need to use a function